operator/config/crd/patches/webhook_in_runtimetasks.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
```

Comment:
CRDs went GA in 1.16, any idea what the plans for kubebuilder are for moving to generating v1 manifests?

Reply:
I have to check plan with the kubebuilder folks, but being those generated files I guess we will get this for free when upgrading the controller-gen dependency/make manifest script